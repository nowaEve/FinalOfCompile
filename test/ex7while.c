int main(){
    int i = 1;
    do{
        i++;
        print i;
    }while(i<=5);
} 