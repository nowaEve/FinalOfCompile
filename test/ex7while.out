24 19 0 5 25 13 0 1 12 16 27 13 11 13 13 11 0 1 1 12 15 -2 13 11 22 15 -1 13 11 0 5 10 7 17 11 21 0